fs/copy_test.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	_ "crypto/sha256"
 
-	"github.com/docker/containerd/fs/fstest"
+	"github.com/docker/containerd/testutil/fstest"
```

Comment:
I think keeping it as `fs/fstest` makes sense. `fstest` is the more specific package name, under `fs`. I agree with Stephen's comment that a general testutil package is not well bounded and easily just sucks in more code.